src/ucp/rndv/rndv.c

```diff
@@ -1984,6 +2004,7 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_data_handler,
         ucp_request_put(rndv_req);
     }
 
+    ucs_profile_range_pop();
```

Comment:
Is it ok that there is no ucs_profile_range_push() in the scope of this range_pop()?